view/vw_qgep_wastewater_structure.py

```diff
@@ -87,7 +87,9 @@ def vw_qgep_wastewater_structure(srid: int,
         LEFT JOIN qgep_od.infiltration_installation ii ON ii.obj_id = ws.obj_id
         LEFT JOIN qgep_od.wastewater_networkelement ne ON ne.obj_id = ws.fk_main_wastewater_node
         LEFT JOIN qgep_od.wastewater_node wn ON wn.obj_id = ws.fk_main_wastewater_node
-        {extra_joins};
+        LEFT JOIN qgep_od.channel ch ON ch.obj_id = ws.obj_id
+        {extra_joins}
+        WHERE ch.obj_id IS NOT NULL;
```

Comment:
am I mixing things, or shouldn't it be the `IS NULL`?

Reply:
```suggestion
        WHERE ch.obj_id IS NULL;
```

correct, thanks